src/lib/store-test.js

```diff
@@ -0,0 +1,222 @@
+const { expect } = require("chai");
```

Comment:
This is not a comprehensive suite of tests - I mostly just started doing TDD in the process of keeping things straight for this new feature.